src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -31,6 +27,10 @@ import { useRadio } from "hooks/useRadio";
 import { useSelector } from "hooks/useSelector";
 import { useUser } from "hooks/useUser";
 import { useVenueId } from "hooks/useVenueId";
+import {
+  SparkleRFQConfig,
+  useSparkleFirestoreConnect,
+} from "hooks/useSparkleFirestoreConnect";
```

Comment:
```suggestion
import { useSelector } from "hooks/useSelector";
import {
  SparkleRFQConfig,
  useSparkleFirestoreConnect,
} from "hooks/useSparkleFirestoreConnect";
import { useUser } from "hooks/useUser";
import { useVenueId } from "hooks/useVenueId";
```

Reply:
What is the difference here? Different sorting order for the hooks among themselves?

Reply:
Sorting the imports alphabetically within that 'group'. The diff view here makes it look way more complex than it is, was just moving the import line up.